docs/docs/integrations/analytics/amplitude.md

```diff
@@ -6,7 +6,7 @@ hide_title: true
 
 ![Amplitude](/img/integrations/amplitude/amplitude-logo.svg)
 
-You can integrate Flagsmith with Amplitude. You can automate AB tests by connecting the Flagsmith platform with
+You can integrate Flagsmith with [Amplitude](https://www.flagsmith.com/blog/feature-flag-analytics-for-users-of-flagsmith-and-amplitude). You can automate AB tests by connecting the Flagsmith platform with
```

Comment:
@MiaLoiselle On a second read, it feels like the first phrase "You can integrate Flagsmith with Amplitude" is actually redundant given the one that follows:

> You can automate AB tests by connecting the Flagsmith platform with Amplitude.

Can you please delete the first phrase and move the link to "Amplitude" to where it appears in the second one?

Reply:
Yes! Thanks for the effort. I will close this PR.